modules/apps/layout/layout-type-controller/layout-type-controller-embedded/src/main/resources/META-INF/resources/layout/view/embedded_js.jspf

```diff
@@ -18,6 +18,11 @@
 	var iframe = A.one('#embeddedIframe');
 
 	if (iframe) {
+		iframe.set(
+			'src',
+			'<%= HtmlUtil.escapeHREF(typeSettingsUnicodeProperties.getProperty("embeddedLayoutURL")) %>'
+		);
```

Comment:
You might want to move this below the `on('load',` part to ensure that the element has the callback before setting the attribute